app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Theme.kt

```diff
@@ -63,7 +63,7 @@ private val DarkColors = darkColorScheme(
     surfaceVariant = md_theme_dark_surfaceVariant,
     onSurfaceVariant = md_theme_dark_onSurfaceVariant,
     outline = md_theme_dark_outline,
-    inverseOnSurface = md_theme_dark_inverseOnSurface,
+    inverseOnSurface = md_theme_light_inverseSurface,
```

Comment:
Please don't modify this file. Also, this is incorrect: why assigning a light-themed color to a dark-themed variable?

Reply:
ohh my bad I was changing the color to see the matching bottom bar color.